Tella/Scenes/Settings/Views/Servers/Uwazi/Step1/UwaziAddServerURLView.swift

```diff
@@ -44,51 +39,52 @@ struct UwaziAddServerURLView: View {
 
                     Spacer()
                         .frame(height: 40)
-
-                    TextfieldView(fieldContent: $serverViewModel.serverURL,
-                                  isValid: $serverViewModel.validURL,
-                                  shouldShowError: $serverViewModel.shouldShowURLError,
-                                  errorMessage: serverViewModel.urlErrorMessage,
+                    TextfieldView(fieldContent: $uwaziServerViewModel.serverURL,
+                                  isValid: $uwaziServerViewModel.validURL,
+                                  shouldShowError: $uwaziServerViewModel.shouldShowURLError,
+                                  errorMessage: uwaziServerViewModel.urlErrorMessage,
                                   fieldType: .url)
                     Spacer()
 
-                    BottomLockView<AnyView>(isValid: $serverViewModel.validURL,
+                    BottomLockView<AnyView>(isValid: $uwaziServerViewModel.validURL,
                                             nextButtonAction: .action,
                                             nextAction: {
-                        self.serverViewModel.checkURL()
-
+                        self.uwaziServerViewModel.checkURL()
                     },
                                             backAction: {
                         self.presentationMode.wrappedValue.dismiss()
                     })
                 } .padding(EdgeInsets(top: 0, leading: 16, bottom: 0, trailing: 16))
 
-                if serverViewModel.isLoading {
+                if uwaziServerViewModel.isLoading {
                     CircularActivityIndicatory()
                 }
             }
         }
         .navigationBarHidden(true)
         .onAppear {
-            guard (serverViewModel.currentServer != nil) else { return }
-            serverViewModel.validURL = true
-//#if DEBUG
-//            serverViewModel.serverURL = "https://horizontal.uwazi.io"
-//#endif
+            #if DEBUG
```

Comment:
@rob-horizontal-iOS It's better to not push the credentials of test

Reply:
Removed this code